### ISSUE TITLE
ellipsizing and terminating punctuation

### ISSUE THREAD

**Issue by jonnybarnes** [tantek/cassis#13]
Your ellipsize functions contains the following punctuation to be cut as well: `@$ -~*()_+[]{}|;,<>`. Should the full stop (`.`) not also be included or would this be an acceptable note:

```
 ending word.… #tag https://permali.nk/abc
```

**Comment by tantek** [comment]
It's deliberately not included, note comment in code a few lines about that with the @$ etc.:

// do not break at %`'"&.!?^ - reasons why to be documented.

The summary is that in such an example (full stop) the break should occur at the *space* after the full-stop not the full-stop itself, or drop an entire word before the full stop. Sot that you get:

ending word. ...

and not:

ending word...

Basically those are all characters that if dropped significantly alter the meaning of the immediately preceding text and thus are kept or dropped with it to avoid ellipsing introducing such an unintended change in meaning.
